src/styles/layout.scss

```diff
@@ -4,31 +4,6 @@
   background-color: var(--amplify-colors-background-primary);
   line-height: var(--amplify-line-heights-medium);
   font-size: var(--amplify-font-sizes-medium);
-  &--home {
```

Comment:
I moved these to spaceship.scss so the spaceship related styling isn't spread out across files

Reply:
🚀